src/main/java/run/halo/app/model/support/HaloConst.java

```diff
@@ -19,6 +19,8 @@
      */
     public final static String USER_HOME = System.getProperties().getProperty("user.home");
 
+    public final static String ENV_HALO_HOME = System.getenv("HALO_HOME");
```

Comment:
没有必这样做，现在可以在配置文件中配置 `workDir`，可以统一配置。

Reply:
刚刚看到了， application.yaml里的workDir有最高优先级。
那现在通过环境变量只算是多了一种配置方式，需不需要去掉？如果从运维方面考虑，部署有特别的需要，可以通过修改环境变量来设置，不需要更改配置代码，有一定的灵活性优势。

Reply:
个人不是太建议，主要是考虑使用者这边的水平。这样可能会导致使用者感到混乱。

Reply:
> 刚刚看到了， application.yaml里的workDir有最高优先级。
> 那现在通过环境变量只算是多了一种配置方式，需不需要去掉？如果从运维方面考虑，部署有特别的需要，可以通过修改环境变量来设置，不需要更改配置代码，有一定的灵活性优势。

起始配置文件中的任何配置，都是可以通过环境变量来配置的，且优先级更高。

Reply:
嗯。确实是可以的。多谢大神指点。　带点号的环境变量设置起来有点特殊: `env "halo.workDir=/tmp/" java -jar halo-1.3.1.jar`

Reply:
> > 刚刚看到了， application.yaml里的workDir有最高优先级。
> > 那现在通过环境变量只算是多了一种配置方式，需不需要去掉？如果从运维方面考虑，部署有特别的需要，可以通过修改环境变量来设置，不需要更改配置代码，有一定的灵活性优势。
> 
> 起始配置文件中的任何配置，都是可以通过环境变量来配置的，且优先级更高。

嗯。确实是可以的。多谢大神指点。　带点号的环境变量设置起来有点特殊: env "halo.workDir=/tmp/" java -jar halo-1.3.1.jar

Reply:
> > 刚刚看到了， application.yaml里的workDir有最高优先级。
> > 那现在通过环境变量只算是多了一种配置方式，需不需要去掉？如果从运维方面考虑，部署有特别的需要，可以通过修改环境变量来设置，不需要更改配置代码，有一定的灵活性优势。
> 
> 起始配置文件中的任何配置，都是可以通过环境变量来配置的，且优先级更高。

嗯。确实是可以的。多谢大神指点。　带点号的环境变量设置起来语法稍微有点特殊: 
`env "halo.workDir=/tmp/" java -jar halo-1.3.1.jar`

Reply:
你可以考虑修改之后，提交上来。

Reply:
> > > 刚刚看到了， application.yaml里的workDir有最高优先级。
> > > 那现在通过环境变量只算是多了一种配置方式，需不需要去掉？如果从运维方面考虑，部署有特别的需要，可以通过修改环境变量来设置，不需要更改配置代码，有一定的灵活性优势。
> > 
> > 
> > 起始配置文件中的任何配置，都是可以通过环境变量来配置的，且优先级更高。
> 
> 嗯。确实是可以的。多谢大神指点。　带点号的环境变量设置起来语法稍微有点特殊:
> `env "halo.workDir=/tmp/" java -jar halo-1.3.1.jar`

环境变量似乎可以将 `.` 号更改为 `_` 来配置。

Reply:
> > > > 刚刚看到了， application.yaml里的workDir有最高优先级。
> > > > 那现在通过环境变量只算是多了一种配置方式，需不需要去掉？如果从运维方面考虑，部署有特别的需要，可以通过修改环境变量来设置，不需要更改配置代码，有一定的灵活性优势。
> > > 
> > > 
> > > 起始配置文件中的任何配置，都是可以通过环境变量来配置的，且优先级更高。
> > 
> > 
> > 嗯。确实是可以的。多谢大神指点。　带点号的环境变量设置起来语法稍微有点特殊:
> > `env "halo.workDir=/tmp/" java -jar halo-1.3.1.jar`
> 
> 环境变量似乎可以将 `.` 号更改为 `_` 来配置，例如：`SPRING_CONFIG_NAME` instead of `spring.config.name`。

nice, 有效 .

Reply:
> 你可以考虑修改之后，提交上来。

已修改，请review -_-